test/functional/versions_controller_test.rb

```diff
@@ -87,27 +87,37 @@ class VersionsControllerTest < ActionController::TestCase
       end
 
       assert_select ".gem__version__date sup", text: "*", count: 1
+
+      assert_select ".t-list__heading", text: /1 versions since January 01, 2000/, count: 1
```

Comment:
"1 version**s**" and not "1 version" is an existing bug that could probably be fixed using https://guides.rubyonrails.org/i18n.html#pluralization in https://github.com/rubygems/rubygems.org/blob/a16630211528eaa14a0e519e44c3022f12925c3f/config/locales/en.yml#L710

Reply:
Well spotted. Would you mind to open PR to fix this please?

Reply:
PR for this is https://github.com/rubygems/rubygems.org/pull/4466